src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stop work");
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer stop work");
```

Comment:
The message in `doWork` method is incorrect. It should print a message indicating that the bulldozer is starting work, such as `"Bulldozer start work"`.